dev-component/wps.cfg

```diff
@@ -1,5 +1,7 @@
 [server]
 outputurl = http://docker-dev03.pcic.uvic.ca:30102/outputs
+maxrequestsize = 65gb
+maxsingleinputsize = 200mb
```

Comment:
What does this do?

Reply:
Or I should have said, why is this necessary?

Reply:
I believe pywps's default maximum input size is 3MB. `maxrequestsize = 65gb` increases that limit to 65GB. I'am not sure what `maxsingleinputsize = 200mb` does, but just mimicked other birds' configurations.

Reply:
> why is this necessary?

This is necessary because the actual use cases will require to input files much larger than 3MB.